src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
```

Comment:
- [ ] Please create tests for all your components and functions. The tests are important tools for the knowledge of developers' careers.

<img width="582" alt="Screen Shot 2022-11-19 at 2 07 39 PM" src="https://user-images.githubusercontent.com/47673311/202863042-a0321c16-03f0-4f5d-b0f6-5c8d51b6cc56.png">